### PR TITLE
add Google Summer of Code 2018 section, and a couple ideas

### PR DESCRIPTION
**Pull request by gregsheremeta**
Changes proposed in this pull request:
add Google Summer of Code 2018 section, and a couple ideas

I confirm that this pull request was submitted according to the [contribution guidelines](https://github.com/oVirt/ovirt-site/blob/master/CONTRIBUTING.md): @gregsheremeta

This pull request needs review by: @jelkosz @mykaul @sandrobonazzola

### CODE REVIEW

File: source/community/activities/summer-of-code.html.md [hunk 1]
```diff
@@ -42,9 +42,34 @@ When students approach you about the idea you listed:
...
+**Expected results:** ovirt-web-ui (already Dockerized) should be adapted to run in Kubernetes. ovirt-web-ui's API should be enhanced or replaced so it works with KubeVirt to start, stop, and console-to virtual machines.
+
+**Knowledge Prerequisite:** JavaScript (react, webpack, etc.), Go (?), Kubernetes
+
+**Mentor:** [Greg Sheremeta](mailto:gshereme@redhat.com)
+
 ## oVirt Ideas for Google Summer of Code 2017
 
-### **Idea:** ovirt4cli
+### **Idea:** ovirt4cli1
```
Comment (rollandf) [inline source/community/activities/summer-of-code.html.md]: Is the "1" addition in purpose?

**Comment by gregsheremeta** [comment]
Nope, typo

**Comment by sandrobonazzola** [comment]
@gregsheremeta  can you please fix the typo and resolve conflicts?

**Comment by gregsheremeta** [comment]
@sandrobonazzola done

**Comment by gregsheremeta** [comment]
can anyone merge?

**Pull request merged**